Pixel_Art_Slideshow/pixel_art_slideshow.py

```diff
@@ -0,0 +1,66 @@
+"""
+Slideshow Example using the Matrix Portal and 64 x 32 LED matrix display
+Written by Melissa LeBlanc-Williams for Adafruit Industries
+Images smaller than 64 pixel width will be aligned alternating left or right
+Press Up button to pause/resume Slideshow
+Press Down button to advance
+"""
+import time
+import board
+from digitalio import DigitalInOut, Pull
+from adafruit_matrixportal.matrix import Matrix
+from adafruit_slideshow import SlideShow, PlayBackDirection, HorizontalAlignment
+from adafruit_debouncer import Debouncer
+
+IMAGE_DURATION = 3
+IMAGE_FOLDER = "/bmps"
+
+# --- Display setup ---
+matrix = Matrix(bit_depth=6)
+display = matrix.display
+
+pin_down = DigitalInOut(board.BUTTON_DOWN)
+pin_down.switch_to_input(pull=Pull.UP)
+button_down = Debouncer(pin_down)
+pin_up = DigitalInOut(board.BUTTON_UP)
+pin_up.switch_to_input(pull=Pull.UP)
+button_up = Debouncer(pin_up)
+
+align_right = True
+auto_advance = True
+
+slideshow = SlideShow(
+    display,
+    None,
+    folder=IMAGE_FOLDER,
+    order=0,
+    auto_advance=False,
+    fade_effect=False,
+    dwell=IMAGE_DURATION,
+    h_align=HorizontalAlignment.RIGHT,
+)
+last_advance = time.monotonic()
+
+
+def advance():
+    # pylint: disable=global-statement
+    global align_right, last_advance
+    align_right = not align_right
```

Comment:
I'm not sure if you want all images to alternate or only 32. If you only want 32 pixel width images to change this, you could load the image first, get the image with slideshow. current_image_name(), load it up, check the width, and then flip align_right only if it's less than 64. Kinda cumbersome, but slideshow doesn't seem to expose much. Or if you're happy with how it works now, then I'm good with it too.

Reply:
When I wrote this, I didn't realize it was going to be a mix of 32 and 64 pixel wide images.

Reply:
happy with how it works now, thanks.

Reply:
Thanks